feature/experimental/isis/ate_tests/isis_metric_style_wide_not_enabled_test/isis_metric_style_wide_not_enabled_test.go

```diff
@@ -0,0 +1,455 @@
+	isisIntfLevel.Enabled = ygot.Bool(true)
+	isisIntfLevel.Passive = ygot.Bool(false)
+	isisIntfLevel.GetOrCreateHelloAuthentication().Enabled = ygot.Bool(true)
+	isisIntfLevel.GetHelloAuthentication().AuthPassword = ygot.String(password)
+	isisIntfLevel.GetHelloAuthentication().AuthType = oc.KeychainTypes_AUTH_TYPE_SIMPLE_KEY
+	isisIntfLevel.GetHelloAuthentication().AuthMode = oc.IsisTypes_AUTH_MODE_MD5
+
+	isisIntfLevelTimers := isisIntfLevel.GetOrCreateTimers()
+	isisIntfLevelTimers.HelloInterval = ygot.Uint32(5)
+	isisIntfLevelTimers.HelloMultiplier = ygot.Uint8(3)
+
+	isisIntfLevel.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
+	isisIntfLevel.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Metric = ygot.Uint32(dutV4Metric)
+	isisIntfLevel.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV6, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
+	isisIntfLevel.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV6, oc.IsisTypes_SAFI_TYPE_UNICAST).Metric = ygot.Uint32(dutV6Metric)
+
+	// Interface afi-safi configs.
+	intf.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
+	intf.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV6, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
+
+	t.Log("Pushing isis config on DUT")
+	gnmi.Replace(t, dut, configPath.Config(), prot)
+}
+
+// configureATE configures the interfaces and isis on ATE.
+func configureATE(t *testing.T, ate *ondatra.ATEDevice) *ondatra.ATETopology {
+	t.Helper()
+	topo := ate.Topology().New()
+	port1 := ate.Port(t, "port1")
+	port2 := ate.Port(t, "port2")
+
+	i1Dut := topo.AddInterface(atePort1attr.Name).WithPort(port1)
+	i1Dut.IPv4().WithAddress(atePort1attr.IPv4CIDR()).WithDefaultGateway(dutPort1Attr.IPv4)
+	i1Dut.IPv6().WithAddress(atePort1attr.IPv6CIDR()).WithDefaultGateway(dutPort1Attr.IPv6)
+
+	i2Dut := topo.AddInterface(atePort2attr.Name).WithPort(port2)
+	i2Dut.IPv4().WithAddress(atePort2attr.IPv4CIDR()).WithDefaultGateway(dutPort2Attr.IPv4)
+	i2Dut.IPv6().WithAddress(atePort2attr.IPv6CIDR()).WithDefaultGateway(dutPort2Attr.IPv6)
+
+	isisDut := i1Dut.ISIS()
+	isisDut.
+		WithAreaID(ateAreaAddress).
+		WithTERouterID(atePort1attr.IPv4).
+		WithNetworkTypePointToPoint().
+		WithWideMetricEnabled(true).
+		WithLevelL2().WithAuthMD5(password).
+		WithAreaAuthMD5(password).
+		WithDomainAuthMD5(password)
+
+	netGrp := i1Dut.AddNetwork(fmt.Sprintf("isis-%d", 1))
+	netGrp.IPv4().WithAddress(ateV4Route)
+	netGrp.ISIS().WithActive(true).WithIPReachabilityMetric(ateV4Metric)
+	netGrp.IPv6().WithAddress(ateV6Route)
+	netGrp.ISIS().WithActive(true).WithIPReachabilityMetric(ateV6Metric)
+
+	t.Log("Pushing config to ATE and starting protocols...")
+	topo.Push(t)
+	topo.StartProtocols(t)
+	return topo
+}
+
+// createFlow returns v4 and v6 flow from atePort2 to atePort1
+func createFlow(t *testing.T, ate *ondatra.ATEDevice, ateTopo *ondatra.ATETopology) []*ondatra.Flow {
```

Comment:
since it returns multiple, rename to createFlows?

Reply:
fixed in latest commit